SEQLinkage/linkage.py

```diff
@@ -0,0 +1,189 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/linkage_analysis.ipynb (unless otherwise specified).
```

Comment:
is this file supposed to replace the `run_linkage.py`

Reply:
Yes. This is the core codes for likelihood calculation.